pkg/generator/vars_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"os"
 	"testing"
 
-	"github.com/alexandrst88/terraform-variables-generator/pkg/utils"
+	"github.com/marceloalmeida/terraform-variables-generator/pkg/utils"
```

Comment:
?

Reply:
It was to be able to perform `go get...` on multiple places and build using my changes, just reverted.